src/transformers/trainer.py

```diff
@@ -223,11 +223,19 @@
 
 
 if is_accelerate_available():
-    from accelerate import Accelerator, skip_first_batches
+    from accelerate import (
+        Accelerator,
+        AutocastKwargs,
+        GradScalerKwargs,
+        InitProcessGroupKwargs,
+        ProfileKwargs,
+        skip_first_batches,
+    )
```

Comment:
some of these kwargs class are quite new, so this will be breaking for users who doesn't have the latest accelerate. The min version of accelerate in transformers is v0.26.0

Reply:
I could move the imports in `_get_accelerator_kwargs_handlers` and check if accelerate version is compatible there?